packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -152,6 +152,7 @@ export class ProfilesCache {
             }
             const allTypes = this.getAllProfileTypes(apiRegister?.registeredApiTypes() ?? []);
             allTypes.push("base");
+            allTypes.push("ssh");
```

Comment:
This was added for extenders calling loadNamedProfile() or other profile fetching APIs for a SSH profile.  possible regression as it previously worked.